Makefile

```diff
@@ -87,6 +88,15 @@ $(PROMQ) promq: $(TOOLS_DIR)
 		rm -rf $$TMP_DIR ;\
 	}
 
+.PHONY: oc
+$(OC) oc: $(TOOLS_DIR)
+	@{ \
+		set -ex ;\
```

Comment:
could you please add `[[ -f $(OC) ]] && exit 0 ; ` so avoid redownloading if it already exists ?

Reply:
Yes, good idea. Done